portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['My favorite color is green.', 'I like playing music (Piano, Guitar, Drums)', "I'm good at bowling.", 'There are 10 kids in my family.'];
```

Comment:
same comment here that Jake made about long lines wrapping. Maybe break this up into multiple lines like
const greetings = 
    ['string 1',
     'string 2',
     'string 3'];